iib/workers/tasks/build.py

```diff
@@ -366,6 +366,10 @@ def _serve_index_registry_at_port(db_path, port, max_tries, wait_time):
                 log.debug('Started the command "%s"', ' '.join(cmd))
                 log.info('Index registry service has been initialized.')
                 return rpc_proc
+            elif 'api.Registry.ListPackages' in output:
```

Comment:
Could you explain the purpose of this? And if it's necessary, couldn't you just combine both statements into one condition?

Reply:
Good point

Reply:
changed